cmd/generate.go

```diff
@@ -592,6 +580,11 @@ func executeTargetSQL(ctx context.Context, config *internal.Config, wd string, t
 	return nil
 }
 
+// generateMissingPermissionStatements generates missing permission statements
+// for the given target and migration connections. This feature is not yet
+// available in pg-schema-diff, but planned.
```

Comment:
Can you link the issue in the comment for reference?

Reply:
There is no issue for it, it's mentioned in their readme.